aspnetcore/fundamentals/host/platform-specific-configuration/samples/2.x/RuntimeStore/Program.cs

```diff
@@ -0,0 +1 @@
+public class Program { public static void Main() {}}
```

Comment:
We've (or at least I have) been adding a CRLF to make GH happy. :smile: ... it isn't really necessary tho.